src/commands/funnies/fridge.ts

```diff
@@ -0,0 +1,12 @@
+import { CommandDefinition } from '../../lib/command';
+import { makeEmbed } from '../../lib/embed';
+import { CommandCategory } from '../../constants';
+
+const FRIDGE_URL = 'https://media.discordapp.net/attachments/898602626436964402/921199183430549524/samsung-family-hub-refrigerator-1_1024x1024.png?width=676&height=676';
```

Comment:
```suggestion
const FRIDGE_URL = 'https://media.discordapp.net/attachments/898602626436964402/921199183430549524/samsung-family-hub-refrigerator-1_1024x1024.png';
```

ngl I can't remember why it's better to remove this bit, but other than this LGTM!